config/locales/en.yml

```diff
@@ -2356,3 +2354,16 @@ en:
           - "(if known) 'We estimate [X] checks were affected by this issue'"
         content: Resolution text
         resolve: Resolve problem
+  feedback:
+   freetext:
+     button: Give feedback on this page
+     heading: What is your feedback on this page?
+     heading_hint: Don't include personal information.
+     button_send: Send
+     button_cancel: Cancel
+     thank_you_message: Thank you for your feedback
+   satisfaction:
+     question: Were you satisfied with this service?
+     thank_you_message: Thank you for your feedback
+     feedback_link_html: <a href="https://eu.surveymonkey.com/r/ccqpublicbeta" target="_blank">Tell us more in our 2 minute survey (opens in new tab)</a>
```

Comment:
We're trying to add `rel="noopener noreferrer"` to all external links. The linter picks up on this for ones in ruby code, but doesn't spot them automatically in en.yml.